columnq/src/query/graphql.rs

```diff
@@ -153,10 +152,10 @@ fn to_datafusion_predicates<'b>(
     }
 }
 
-pub fn query_to_df(
+pub async fn query_to_df(
     dfctx: &datafusion::execution::context::SessionContext,
     q: &str,
-) -> Result<Arc<datafusion::dataframe::DataFrame>, QueryError> {
+) -> Result<datafusion::dataframe::DataFrame, QueryError> {
```

Comment:
@houqp do you know why we used `Arc` before?

I run into the following error and removing `Arc` seems to resolve it. Do you think it's the correct approach?
```
df.collect() now returns "error[E0507]: cannot move out of an Arc"
move occurs because value has type datafusion::dataframe::DataFrame, which does not implement the Copy trait
```

Reply:
Yes, in general, if we can get the code to compile without Arc, then we should avoid it.

Reply:
this was required because previous versions of datafusion's SessionContext returns `Arc` wrapped dataframe. This api has been changed recently.

Reply:
right, found it
https://github.com/apache/arrow-datafusion/pull/4624